src/test/java/com/karankumar/bookproject/backend/service/ReadingGoalServiceTest.java

```diff
@@ -43,4 +56,22 @@ void expectSavingGoalToOverwriteExistingGoal(@NotNull @Autowired ReadingGoalServ
         ReadingGoal actual = goalService.findAll().get(0);
         assertThat(newReadingGoal).isEqualToComparingFieldByField(actual);
     }
+
+    @Test
+    void expectDeleteExistingGoal() {
+        // given
+        assertThat(goalService.count()).isOne();
```

Comment:
As with the other first assertion, could you please change this to an assumption?